scripts/vendor/siemaWithDots.js

```diff
@@ -0,0 +1,45 @@
+// slightly modified version of a codepen by Pawel Grzybek
+// at https://codepen.io/pawelgrzybek/pen/boQQWy
+
+// extend a Siema class by two methods
+// addDots - to create a markup for dots
+// updateDots - to update classes on dots on change callback
+class SiemaWithDots extends Siema {
```

Comment:
this is mostly Pawel's code, but as you can see, it's fairly simple... it'd be a little more complicated if it was done outside of siema, which is what we were trying to do initially